docs/mkdocs/docs/api/macros/json_diagnostic_positions.md

```diff
@@ -50,18 +49,25 @@ When the macro is not defined, the library will define it to its default value.
 
 !!! note "CMake option"
 
-    Diagnostic messages can also be controlled with the CMake option
+    Diagnostic positions can also be controlled with the CMake option
     [`JSON_Diagnostic_Positions`](../../integration/cmake.md#json_diagnostic_positions) (`OFF` by default)
     which defines `JSON_DIAGNOSTIC_POSITIONS` accordingly.
 
+!!! note "Availability"
+
+    Diagnostic positons are only available if the value was created by the [`parse`](../basic_json/parse.md) function.
```

Comment:
s/positons/positions/

Reply:
Fixed in 82151089113db79447d3f3a017fcfbfdbb6fd231.